packages/core/src/utils/combat-simulator.ts

```
@@ -0,0 +1,147 @@
+      this.BETA_SMALL -
+      (this.BETA_SMALL - this.BETA_LARGE) * ((Math.tanh((totalTroops - this.C0) / this.DELTA) + 1) / 2)
+    );
+  }
+
+  public static simulateBattle(
+    attacker: Army,
+    defender: Army,
+    biome: Biome,
+  ): { attackerDamage: number; defenderDamage: number } {
+    const totalTroops = attacker.troopCount + defender.troopCount;
+    const betaEff = this.calculateEffectiveBeta(totalTroops);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)